applications/holochat/requirements.txt

```diff
@@ -1,4 +1,4 @@
-langchain==0.1.11
+langchain==0.2.5
```

Comment:
based on our discussions https://github.com/nvidia-holoscan/holohub/pull/981#discussion_r2192471994 this langchain change suggested by dependabot is not a feasible way, is it possible to turn off the auto correction? @nvidia-holoscan/holohub